synth.py

```diff
@@ -0,0 +1,28 @@
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import synthtool as s
+from synthtool import gcp
+
+common = gcp.CommonTemplates()
+
+# ----------------------------------------------------------------------------
+# Add templated files
+# ----------------------------------------------------------------------------
+templated_files = common.py_library(unit_cov_level=100, cov_level=100)
```

Comment:
We recently ditched the magic threshold numbers for individual unit test runs in other libraries, although having it here at 100 should be fine.

Is there a chance that a future fix would have to branch based on the Python version, causing the individual coverage check to start failing?

Reply:
I removed the threshold values as it wasn't being used. (The noxfile for this repo is handwritten). The existing coverage threshold for this library is [85%](https://github.com/googleapis/google-api-python-client/blob/master/tox.ini#L24) so I used that value in unit test session for `noxfile.py`.